src/tools/annotation/RectangleRoiTool.js

```diff
@@ -478,7 +478,7 @@ function _getUnit(modality, showHounsfieldUnits) {
 function _createTextBoxContent(
   context,
   isColorImage,
-  { area, mean, stdDev, min, max, meanStdDevSUV },
+  { area = 0, mean = 0, stdDev = 0, min = 0, max = 0, meanStdDevSUV = 0 } = {},
```

Comment:
change to 

context,
  isColorImage,
  rectangleRoiContent ={}
  ...) {
  const { area = 0, mean = 0, stdDev = 0, min = 0, max = 0, meanStdDevSUV = 0 } = rectangleRoiContent,
  .....